app-frontend/src/app/pages/projects/detail/exports/exports.html

```diff
@@ -0,0 +1,49 @@
+<div class="list-group">
+    <rf-export-item
+        export="export"
+        ng-repeat="export in $ctrl.exportList track by export.id">
+    </rf-scene-item>
+  </div>
+  <!-- Export List -->
+
+  <!-- Pagination -->
+  <div class="list-group text-center"
+       ng-show="!$ctrl.loading && $ctrl.lastExportResult && $ctrl.lastExportResult.count > $ctrl.lastExportResult.pageSize && !$ctrl.errorMsg">
+    <ul uib-pagination
+        items-per-page="$ctrl.lastExportResult.pageSize"
+        total-items="$ctrl.lastExportResult.count"
+        ng-model="$ctrl.currentPage"
+        max-size="4"
```

Comment:
We can probably go a bit higher - I just tried with 10 buttons and it looks fine at single digits
![image](https://user-images.githubusercontent.com/4392704/27547715-d1a46ea0-5a64-11e7-823a-4e537b726681.png)
ignore the numbers. I just copied and pasted html in the page because I don't have that many exports.
6 is probably a more reasonable number